_pages/enforcement.md

```diff
@@ -4,18 +4,20 @@ title: "Enforcement"
 permalink: /enforcement
 ---
 
-# Code of Conduct Enforcement
-
-## tl;dr
-
-  1. Tell someone "hey, that's not ok".
-  2. That person should apologize immediately and correct the behavior.
-  3. If this doesn't happen, isn't safe to do, or is not satisfactory, message admins.
-  4. Admins will take appropriate action in a timely manner, prioritizing those who were hurt.
+# Code of Conduct Enforcement Philosophy
 
 ## Introduction
 
-No matter how kind or nice people think they are, at some point, someone's gonna say Some Shit:tm: and things are going to go awry. This document is meant as a writeup and guideline holding core values about what this community considers the best approach to resolving these conflicts in a kind and positive way, as much as possible. It is paired with the community's [Code of Conduct](/code-of-conduct). It is a living document, informed by new problems, new conflicts, and always strives to help make the WeAllJS community successful while including the voices of those who are too often not heard.
+No matter how kind or nice people think they are, at some point, someone's gonna
+say Some Shit:tm: and things are going to go awry. This document is meant as a
```

Comment:
Is the profanity okay? The document is fairly informal, but I'm not sure exactly how informal it should be.